jstz_node/src/rollup.rs

```diff
@@ -17,11 +17,15 @@ pub struct ValueError {
 
 #[derive(Deserialize)]
 #[serde(untagged)]
+#[derive(Debug)]
```

Comment:
Nit: Multiple derives

```suggestion
#[derive(Debug, Deserialize)]
#[serde(untagged)]
```